client/src/components/header/header.vue

```diff
@@ -3,10 +3,21 @@ import { useChannelStore } from '../../stores/channel';
 import { default as Button } from '../generic-button/generic-button.vue';
 import GameInfo from '../game-info/game-info.vue';
 import { resetApp } from '../../main';
-import { computed } from 'vue';
+import { computed, onUpdated, reactive, ref } from 'vue';
 
 const channelStore = useChannelStore();
-const repoURL = 'https://github.com/aeternity/state-channel-demo';
+const repoUrl = 'https://github.com/aeternity/state-channel-demo';
+const explorerUrl = ref('');
+
+const channelWrapper = reactive({
+  channelStore,
+});
+
+onUpdated(() => {
+  if (channelWrapper.channelStore.channel?.channelConfig?.responderId) {
+    explorerUrl.value = `https://testnet.aenalytics.org/accounts/${channelWrapper.channelStore.channel?.channelConfig?.responderId}`;
+  }
```

Comment:
When we're working on local network, this won't work properly. In those cases, I propose the following:

`<NODE_URL>/v2/channels/<CHANNEL_ID>`
example:
http://localhost:3013/v2/channels/ch_23GydFRjjLMDhw5bf3xMgPePzFUpC7GtNC4NBLW3ktUY2NoVNR

![image](https://user-images.githubusercontent.com/45261205/187890570-8e4f01c9-117d-44a6-bd52-3473103bc102.png)